SPECS/systemd/systemd.spec

```diff
@@ -469,6 +469,8 @@ Requires:       python3dist(cryptography)
 Recommends:     python3dist(pillow)
 %endif
 
+BuildRequires:  python3dist(pyflakes)
```

Comment:
Move this `BuildRequires` up to where the other `BuildRequires:` are. IIRC there isn't a concept of "subpackage" BuildRequires, since there is one overall build step that occurs during rpmbuild. And keeping BuildRequires down here ends up hiding this specific BuildRequires in the overall spec.

Additionally, is pyflakes needed for general operation or just for testing? If needed just for testing, you could also consider wrapping the BuildRequires in a `%if 0%{?with_check}` so it only is installed when testing occurs